### PR TITLE
DOC: ReStructuredText README for PyPI

### LINKED ISSUE #35: ReStructuredText README for PyPI
It'd be nice if the README were .rst instead of .md, for [PyPI](https://pypi.python.org/pypi/glmnet/2.0.0).

I think one could just 

`pandoc --from=markdown --to=rst --output=README.rst README.md`

### PR DESCRIPTION
**Pull request by wlattner**
Closes #35.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+The API follows the conventions of
+`Scikit-Learn <http://scikit-learn.org/stable/>`__, so it is expected to
+work with tools from that ecosystem.
+
+Installation
+------------
+
+``glmnet`` depends on numpy, scikit-learn and scipy. A working Fortran
+compiler is also required to build the package, for Mac users,
+``brew install gcc`` will take care of this requirement.
```
Comment (beckermr) [inline README.rst]: You might also mention the conda forge package which works now.

**Comment by beckermr** [comment]
It looks like you missed changing the manifest.in file. I think setuptools includes rst by default? Otherwise I do not see how the tests would pass.

**Comment by wlattner** [comment]
(╯°□°)╯︵ ┻━┻

**Comment by beckermr** [comment]
LGTM!

**Pull request merged**